starknet-replay/src/profiler/analysis.rs

```diff
@@ -165,10 +169,10 @@ mod tests {
             .unwrap_or_else(|_| panic!("Unable to read file {sierra_program_json_file}"));
         let sierra_program_json: serde_json::Value = serde_json::from_str(&sierra_program_json)
             .unwrap_or_else(|_| panic!("Unable to parse {sierra_program_json_file} to json"));
-        let contract_class: SierraContractClass =
-            serde_json::from_value::<SierraContractClass>(sierra_program_json).unwrap_or_else(
-                |_| panic!("Unable to parse {sierra_program_json_file} to SierraContractClass"),
-            );
+        let contract_class: ContractClass =
+            serde_json::from_value::<ContractClass>(sierra_program_json).unwrap_or_else(|_| {
```

Comment:
Surely the annotation on the let binding should obviate the need for the turbofish here?